### PR TITLE
docs/15495-padding-ordinal 

### LINKED ISSUE #15495: Docs: min/maxPadding does not explain relation with xAxis.ordinal option
#### Expected behaviour
API should explain that `Axis.ordinal` option takes precendence over`Axis.minPadding` and `Axis.maxPadding` options and ignores them. For `Axis.ordinal` users add a note about `Axis.overscroll` option. Add the explanation under `min/maxPadding` options (Stock only)

#### Actual behaviour
Missing explanation

#### Product version
Highcharts Stock

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed #15495, added explanation about paddings with an ordinal axis.

### CODE REVIEW

File: ts/Core/Axis/Axis.ts [hunk 1]
```diff
@@ -1938,6 +1938,9 @@ class Axis {
          * of the plot area. When the axis' `max` option is set or a max extreme
          * is set using `axis.setExtremes()`, the maxPadding will be ignored.
          *
+         * @productdesc {highstock}
+         * When the axis `ordinal: true`, the `maxPadding` is ignored.
```
Comment (raf18seb) [inline ts/Core/Axis/Axis.ts:1942]: I think we could improve this sentence. What about the same what you used in `xAxis.ordinal`?
For example:
```
* For the ordinal axis, the `minPadding` and `maxPadding` are ignored -
* use the `overscroll` instead.
```

**Review by pawellysy — APPROVED** [review]
See @raf18seb  comment. Apart from that, looks good!

**Review by raf18seb — APPROVED** [review]
Thanks!

**Review by pawelfus — APPROVED** [review]
👍

**Pull request merged**